CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.1.2] - 2019-07-22
```

Comment:
In response to https://github.com/giantswarm/architect-orb/pull/14#issuecomment-514150383:

> @kopiczko I bump version in a commit immediately after the release, and make release after merge to master so need approval for the PR. Release notes / changelog update has already been included in the PR. All of this follows release process docs https://github.com/giantswarm/architect-orb#releases

Then this should stay Unreleased and we should change that to 0.1.2 and add the date with the follow up PR. I don't mind releasing this within this PR but we should do it either way IMO. Yes this is nitpicky but let's build some good habits.